mock/v1.2/channels/lib/channelService-1_2.js

```diff
@@ -40,13 +40,15 @@ class Fdc3CommandExecutor1_2 {
     if (config.notifyAppAOnCompletion) {
       await this.notifyAppAOnCompletion(config.testId);
     }
-
   }
 
-  async joinUserChannelOne() {
-    const channels = await window.fdc3.getSystemChannels();
-    await window.fdc3.joinChannel(channels[0].id);
-    return channels[0];
+  async joinRetrievedUserChannel(channelId) {
+    const systemChannels = await window.fdc3.getSystemChannels();
+    const joinedChannel = systemChannels.find((c) => c.id === channelId);
+    if(joinedChannel){
+      await window.fdc3.joinChannel(channelId);
```

Comment:
so it is still joining the first channel from the array in the tests -- but this time it grabs the id of that channel and sends it off to the mock app, and in the mock app, instead of joining the first channel in the array, this time it joins by id. Does that make sense